xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-test/xwiki-platform-ckeditor-test-docker/src/test/it/org/xwiki/ckeditor/test/ui/QuickActionsIT.java

```diff
@@ -70,25 +88,35 @@ private static void createAndLoginStandardUser(TestUtils setup)
     
     @Test
     @Order(1)
-    void heading1(TestUtils setup, TestReference testReference) throws Exception
+    void heading1AndParagraph(TestUtils setup, TestReference testReference) throws Exception
     {
         WYSIWYGEditPage editPage = setup.gotoPage(testReference).editWYSIWYG();
         CKEditor editor = new CKEditor("content").waitToLoad();
         
         RichTextAreaElement textArea = editor.getRichTextArea();
 
-        // Run the action
-        QuickActionsDropdown qa = editor.openQuickActionsDropdown();
+        // Switch to another style
+        textArea.sendKeys("/");
+        AutocompleteDropdown qa = new AutocompleteDropdown();
         textArea.sendKeys("hea");
-        qa.waitForQuickActionSelected("Heading 1");
+        qa.waitForItemSelected("Heading 1");
         textArea.sendKeys(Keys.ENTER);
-        qa.waitForQuickActionSubmitted();
+        qa.waitForItemSubmitted();
         
         // Write some text
         textArea.sendKeys(TEST_TEXT);
         
         assert textArea.getContent().contains("<h1>" + TEST_TEXT + "<br></h1>");
-        editPage.clickSaveAndView();
+        
+        // Switch back to paragraph
+        textArea.sendKeys("/");
+        qa = new AutocompleteDropdown();
+        textArea.sendKeys("parag");
+        qa.waitForItemSelected("Paragraph");
+        textArea.sendKeys(Keys.ENTER);
+        qa.waitForItemSubmitted();
+        
+        assert textArea.getContent().contains("<p>" + TEST_TEXT + "<br></p>");
```

Comment:
I think we prefer the JUnit assertions (``Assertions.assertTrue``) because the behaviour of ``assert`` can be disabled, thus making the tests pass.

Reply:
In general `assert` is generally more something mean to be used at runtime, as a checkpoint (but that's not really something we do in XWiki).